src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -666,6 +680,7 @@ def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
 
     # Deleting the azure-arc agents
     utils.delete_arc_agents(release_namespace, kube_config, kube_context, configuration)
+    utils.check_delete_job(configuration, consts.Arc_Namespace)  # Checks for the completion/absence of delete job
```

Comment:
We already wait for 'azure-arc' namespace termination in utils.delete_arc_agents. This check is irrelevant here

Reply:
This is different. If you remember, we get issues sometimes, where namespace is deleted, but delete job is stuck. 
Can you have a look at this function **_check_delete_job()_**: https://github.com/alphaWizard/azure-cli-extensions/blob/0b06cf230e5032b2b6eb9c34a90d0f3ae6f6ec00/src/connectedk8s/azext_connectedk8s/_utils.py#L524

Reply:
We can discuss on call later, if required.

Reply:
Link to job yaml definition: https://github.com/Azure/ClusterConfigurationAgent/blob/4aad9e9cf04897ffb4f42b0dd5b620b0ac07f83d/charts/azure-arc-k8sagents/templates/hooks.yaml#L14

Reply:
The delete job pod runs in the azure-arc namespace itself. So, the 'azure-arc' ns should not go away until the pod has deleted the CRDs. Did you see otherwise on some setup?